packages/fcl/src/transaction/index.js

```diff
@@ -84,23 +114,34 @@ const spawnTransaction = transactionId => {
 
 /**
  * Provides methods for interacting with a transaction
- * 
+ *
  * @param {string} transactionId - The transaction ID
+ * @param {object} [opts] - Optional parameters
+ * @param {number} [opts.pollRate=2500] - Polling rate in milliseconds
+ * @param {number} [opts.txNotFoundTimeout=12500] - Timeout in milliseconds for ignoring transaction not found errors (do not modify unless you know what you are doing)
```

Comment:
Thinking 12500 is good because target because with pollRate of 2500, it guarantees last poll happens >~10000ms which I think we discussed 10s as the target here, but LMK.